pkg/cmd/server/api/validation/validation_test.go

```diff
@@ -206,3 +211,41 @@ MCUXgckTpKCuGwbJk7424Nb8bLzf3kllAiA5mUBgjfr/WtFSJdWcPQ4Zt9KTMNKD
 EUO0ukpTwEIl6wIhAMbGqZK3zAAFdq8DD2jPx+UJXnh0rnOkZBzDtJ6/iN69AiEA
 1Aq8MJgTaYsDQWyU/hDq5YkDJc9e9DSCvUIzqxQWMQE=
 -----END RSA PRIVATE KEY-----`)
+
+func TestLDAPSyncConfigFixtures(t *testing.T) {
+	fixtures := []string{}
+
+	// build a list of common configurations for all schemas
+	schemas := []string{"rfc2307", "ad", "augmented-ad"}
+	for _, schema := range schemas {
```

Comment:
> What reads better:
> 
> whitelisting specific files in a folder not specifically holding things we care about
> blacklist the 30 other things in the folder that we don't care about

Didn't realize we had combined all the artifacts.  I'd still prefer to check them by default, but ignore things that aren't in our scheme.  That's currently hard to do, so a whitelist is alright for now.

Generally, if you have a test like this, you want it to expand by default and make people think about opt-ing out.  That's why `examples_test` requires an explicit excludes.